# HG changeset patch
# User Rik <rik@octave.org>
# Date 1372715211 25200
#      Mon Jul 01 14:46:51 2013 -0700
# Node ID 83425e536ce12083461ee06caad4b6ceb37fae66
# Parent  6ce16c7300c74e67449d0cb601a5b433ee8a418e
NEWS: add stemleaf to list of new functions for 3.8 (bug #39386)

* NEWS: add stemleaf to list of new functions for 3.8 (bug #39386)

diff --git a/NEWS b/NEWS
--- a/NEWS
+++ b/NEWS
@@ -179,21 +179,21 @@ Summary of important user-visible change
       base64_decode               ellipke         lines      
       base64_encode               erfcinv         polyeig                   
       betaincinv                  erfi            readline_re_read_init_file
       built_in_docstrings_file    expint          readline_read_init_file 
       cmpermute                   findfigs        rgbplot               
       cmunique                    fminsearch      save_default_options  
       colorcube                   gallery         shrinkfaces           
       copyobj                     gco             splinefit             
-      dawson                      hdl2struct      strjoin               
-      dblist                      history_save    struct2hdl          
-      debug_jit                   importdata      tetramesh           
-      doc_cache_create            iscolormap      waterfall           
-      ellipj                      jit_enable
+      dawson                      hdl2struct      stemleaf           
+      dblist                      history_save    strjoin             
+      debug_jit                   importdata      struct2hdl          
+      doc_cache_create            iscolormap      tetramesh           
+      ellipj                      jit_enable      waterfall  
 
  ** Deprecated functions.
 
     The following functions were deprecated in Octave 3.4 and have been
     removed from Octave 3.8.
                                            
       autocor    dispatch              is_global    setstr
       autocov    fstat                 krylovb      strerror
